src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -817,17 +817,24 @@ def __try_add_error(error_list, detail):
         error_list.insert(0, detail)
         return True
 
-    def __set_errors_json(self, error_count_by_operation, errors_by_operation):
+    def __set_errors_json(self, error_count_by_operation, errors_by_operation, is_status_truncated=False):
         """ Compose the error object json to be added in 'errors' in given operation's summary """
+        code = Constants.PatchOperationTopLevelErrorCode.SUCCESS if error_count_by_operation == 0 else Constants.PatchOperationTopLevelErrorCode.ERROR
+
         if error_count_by_operation == 1 and errors_by_operation[0]['code'] == Constants.PatchOperationErrorCodes.INFORMATIONAL:    # special-casing for single informational messages
             message = errors_by_operation[0]['message']
             errors_by_operation = []
-            error_count_by_operation = 0
```

Comment:
Why is this changing?

Reply:
i reverted the changes back to what it was before, so i created a new set_truncated_errors method to separate the logic becuase currently logic reset the details[] which wipe out the truncation details